ui/app/templates/rule_scheduled_changes.html

```diff
@@ -54,13 +54,13 @@
     <h3 class="panel-title">
       <div style="float: right">
         <i ng-if=" scheduled_changes_rules_count && $first && (currentPage == 1)">Current</i>
-        <button class="btn btn-xs btn-primary" ng-click="signoff(sc)"
-                ng-show="! isEmpty(sc['required_signoffs']) && ! sc['signoffs'].hasOwnProperty(current_user)">
-            Signoff as...
+        <button ng-show="!scheduled_changes_rules_count && (! isEmpty(sc['required_signoffs']) &&
```

Comment:
It would be better to use sc_id here -- that's the variable that actually determines whether or not we're on the history page.